python/lsst/ts/watcher/rules/mtdome_subsystem_enabled.py

```diff
@@ -99,11 +149,24 @@ def compute_alarm_severity(
         You may return `NoneNoReason` if the alarm state is ``NONE``.
         """
 
+        if hasattr(data, "faultCode"):
+            self.subsystem_state = SubsystemState(
+                state=EnabledState(data.state), fault_code=data.faultCode
+            )
+            self.log.debug(f"{self.subsystem_state.fault_code=!r}")
+        if hasattr(data, "summaryState"):
+            self.may_raise = State(data.summaryState).name in self.config.csc_state
+            self.log.debug(f"{State(data.summaryState).name=!r}, {self.may_raise=}")
+
+        if not self.may_raise:
+            return NoneNoReason
+
         return (
             NoneNoReason
-            if not data.faultCode
+            if not self.subsystem_state or not self.subsystem_state.fault_code
             else (
                 AlarmSeverity.CRITICAL,
-                f"Dome az rotation state {EnabledState(data.state)!r}: {data.faultCode}.",
+                f"Dome az rotation state {self.subsystem_state.state!r}: "
```

Comment:
You generalized the alarm to any subsystem, you probably need to update the text here to log the appropriate subsystem no? Maybe the configuration will require a provided subsystem name?

Reply:
Good catch and suggestion.